### DISCUSSION TITLE
How can I install Google Chrome?

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/triage#128] [Q&A] [answered]
I use Google Chrome on my other devices, and would like to use it on elementary OS. How do I install it?

**Comment by cassidyjames** [comment, marked as answer]
Google only [officially supports Ubuntu, Debian, openSUSE, and Fedora](https://support.google.com/chrome/answer/95346#zippy=%2Clinux), and does not develop Chrome for elementary OS. However, you have a few options:

## 1. Use Web and Firefox

elementary OS comes with Web (called “Epiphany” in elementary OS 5.1), the open source web browser developed by the GNOME Foundation. It is based on WebKit, the same engine powering Safari on macOS, iOS, and iPad OS—and the same engine on which Google Chrome was originally based. It is widely compatible with HTML, CSS, and JavaScript web standards, including most websites. Web also supports **Firefox Sync**, meaning it can synchronize your bookmarks, logins and passwords, and even open tabs between other devices that have Web or Firefox installed.

- On **elementary OS**, open Web, navigate to Preferences → Sync, and sign into a Firefox Sync account to start syncing your browser data.
- For **other devices and operating systems** including macOS, Windows, iOS, and Android, you can install and use Firefox and sign into the same Firefox Sync account and get a largely similar experience to using Google Chrome on multiple devices.

## 2. Install Chromium

Chromium is the open source core of Google Chrome. In essence, it is Google Chrome without Google's trademarks and any extra, proprietary software Google includes in Chrome. For most uses, it is identical to using Google Chrome.

- On **elementary OS 5.1 and older**, Chromium is in AppCenter; simply open AppCenter, search for it, and install it.
- On **elementary OS 5.1 and newer**, Chromium can be installed as a [Flatpak from Flathub](https://flathub.org/apps/details/org.chromium.Chromium).

Google may be removing Google Account support (including sync) from Chromium in the future. If that feature is required, you may be interested in installing the Ubuntu version of Google Chrome.

## 3. Install Google Chrome for Ubuntu

Since elementary OS is generally compatible with Debian and Ubuntu packages, you can install the Ubuntu version of Google Chrome. Visit [google.com/chrome](https://google.com/chrome) download the `64 bit .deb (For Debian/Ubuntu)` file for Google Chrome and save it to your Downloads folder. You can then install it from the Terminal app, or with a third-party app like [Eddy](https://appcenter.elementary.io/com.github.donadigo.eddy/):

- To use **Terminal**, open Terminal and navigate to your Downloads folder (e.g. with `cd ~/Downloads`). Type `sudo apt install ./` and then the name of the downloaded `.deb` file, e.g. `sudo apt install ./google-chrome-stable_current_amd64.deb`. Press Enter and you will be prompted to type your user password. Type it, press Enter, and Google Chrome will be installed.

- To use **Eddy**, first install [Eddy from AppCenter](https://appcenter.elementary.io/com.github.donadigo.eddy/). Then just click on the downloaded Google Chrome `.deb` file and follow the prompts in Eddy to install it.

:warning: **Note: Installing Google Chrome adds a Google software repository to your installation of elementary OS.** This means future versions of Google Chrome will show up alongside your OS updates, but also means that Google theoretically has root access to your device. It also means you could run into issues updating if the Google repository is inaccessible for some reason.

**Reply by kem-a** [reply]
## 4. Install Microsoft Edge
Many don't know that, but Microsoft Edge is available on Linux. It's based on Chromium, is very fast, feature rich, supports web apps (boo Firefox boo) and even Chrome extensions. It's currently available on *Dev Channel*, but actually is very stable and usable as daily browser. Someone who dislikes Google, but wants good fast browser this is very good option.

You can install from here [https://www.microsoftedgeinsider.com/en-us/](https://www.microsoftedgeinsider.com/en-us/)

@cassidyjames, as much as I wanted to love Epiphany it still has too many performance and compatibility issues, for instance scrolling on Reddit is unbearable.
